akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/internal/ClusterShardingImpl.scala

```diff
@@ -308,12 +307,12 @@ import akka.util.Timeout
   override def tell(msg: M): Unit =
     shardRegion ! ShardingEnvelope(entityId, msg)
 
+  @InternalStableApi
```

Comment:
Do you need both the entry point and the `EntityPromiseRef.ask` that you then call?

Reply:
Good catch! Thanks, Johan.